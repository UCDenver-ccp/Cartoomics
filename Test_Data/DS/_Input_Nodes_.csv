source|target
insulin receptor|IRS1
IRS1|PTEN (human)
PTEN (human)|AKT3
AKT3|Glucose metabolism
AKT3|response to oxidative stress
APP|response to oxidative stress
APP|tumor necrosis factor isoform 1 cleaved 1
tumor necrosis factor isoform 1 cleaved 1|IRS1
APP|insulin receptor
