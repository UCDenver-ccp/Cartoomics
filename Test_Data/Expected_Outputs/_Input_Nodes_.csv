source|target|source_label|target_label
label1|label2|label1|label2
label2|label3|label2|label3
